packages/redux/src/checkout/draftOrders/types/state.types.ts

```diff
@@ -0,0 +1,40 @@
+import type { BlackoutError, DraftOrder } from '@farfetch/blackout-client';
+
+export type DraftOrderNormalized = Omit<DraftOrdersEntries, 'entries'> & {
```

Comment:
This does not seem correct, did you mean `DraftOrdersNormalized`? Even so, the DraftOrdersEntries type specified here is not used anywhere, is this correct?

Reply:
That's right, I meant `DraftOrdersNormalized`, adjustment made. So I need normalized because it's an API response. 
I use it to get all the draft orders.

Reply:
I agree, I'll make the adjustment

Reply:
Done 👍🏻